.ado/azure-pipelines.yml

```diff
@@ -116,6 +125,10 @@ jobs:
         parameters:
           relative_directory: $(relative_directory)
 
+      # - ${{ if eq(variables[relative_directory], 'macos')}}:
```

Comment:
?

Reply:
YML is hard to verify. locally so most likely you'll see a lot of small commits to try and debug the yml till it works

Reply:
Testing. Will mark this PR as a draft until I get it down